docs/Configuration.md

```diff
@@ -343,6 +343,12 @@ An array of file extensions your modules use. If you require modules without spe
 
 If you are using TypeScript this should be `["js", "jsx", "json", "ts", "tsx"]`, check [ts-jest's documentation](https://github.com/kulshekhar/ts-jest).
 
+### `moduleHiddenFiles` [array<string>]
```

Comment:
Not too sure about this name. `watchHiddenFiles` perhaps?

Also not sure if we need an option, or just do `.gitignore` and `.hgignore` with no customization option

Reply:
> Not too sure about this name. `watchHiddenFiles` perhaps?

it would be nicer one 👍

> Also not sure if we need an option, or just do `.gitignore` and `.hgignore` with no customization option

I'd like to go with the current direction because by making this feature customizable, users can turn off this feature if they want.